validator/keymanager/imported/keymanager.go

```diff
@@ -46,6 +46,8 @@ const (
 type KeymanagerOpts struct {
 	EIPVersion string `json:"direct_eip_version"`
 	Version    string `json:"direct_version"`
+	// Comma separated hexstrings
+	DisabledPublicKeys string `json:"disabled_public_keys"`
```

Comment:
I had that originally as `[][]byte`, but this was breaking protobuf definition of keymanager config.

Reply:
Opted to have a comma separated hexstrings then.

Reply:
@fabdarice [][]byte would be best, what does it break in particular, can you link us to a file or function ?

Reply:
AFAIK @rauljordan it was breaking this pb message https://github.com/prysmaticlabs/prysm/blob/master/proto/validator/accounts/v2/web_api.proto#L146

```
    map<string, string> keymanager_config = 3;

```

Reply:
@fabdarice whenever we use that pb, we can just convert from [][]byte to a comma-separated string. Better to do that than add string manipulation logic elsewhere